client/modules/users/authReducer.js

```diff
@@ -32,7 +32,7 @@ export const clearFlags = () => ({type: CLEAR_FLAGS})
 
 export const setProfile = user => ({
   [CALL_API]: {
-    endpoint: 'users',
+    endpoint: 'users/me',
```

Comment:
I don't know about reusing the me endpoint for this, seems a bit inconsistent with resource/:resourceId for updates too?

Reply:
They are supposed to represent two different resources. Your own profile, accessible to everyone, and an arbitrary user, accessible only to admins. Resources don't have to map one to one with database objects...or documents, or whatever they're called in mongo.

Now that I look at it, `userRouter.route('/users/:userId').get(users.getById)` should be admin only also. No reason to let normal users fetch other users' data, especially with sequential ids.

But if you don't like the idea, I could solve #350 otherwise.

Reply:
Ah yeah sorry, and if it's only for updating your own profile it's pointless to pass the id.

Good catch on the user get.